plugins/reports/counter/classes/releases/CounterRelease3_0.inc.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/releases/CounterRelease3_0.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterRelease3_0
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, release 3.0
+ */
+
+import('plugins.reports.counter.classes.releases.CounterRelease');
+
+class CounterRelease3_0 extends CounterRelease {
```

Comment:
I struggled with this.  Yes, everything is ultimately a CounterReport, but it is implemented in a cross-product of Releases and Reports.  JR1 has a particular definition, for example, but is not only packaged differently in Release 3.0 than it is in Release 4.1, it is also potentially *formulated* differently.

I like the idea of everything being a CounterReport.  You propose that class is then implemented as a CounterRelease*NN*, and that release is then implemented as a Report*NNXX*?